cmd/builder/README.md

```diff
@@ -152,6 +152,26 @@ ocb --skip-generate --skip-get-modules --config=config.yaml
 ```
 to only execute the compilation step.
 
+### Avoiding the use of a new go.mod file
+
+There is an additional option that controls one aspect of the build
+process, which specifically allows skipping the use of a new `go.mod`
+file.  When the `--skip-new-go-module` command-line flag is supplied,
```

Comment:
I would keep it a bit simpler
```suggestion
You can optionally skip creating a new `go.mod` file.
When the `--skip-new-go-module` command-line flag is supplied,
```

It would also be nice to motivate this: when would an end-user be interested in using `--skip-new-go-module`?